react-native/react/actions/tracker.js

```diff
@@ -292,36 +310,39 @@ function serverCallMap (dispatch: Dispatch, getState: Function): CallMap {
       dispatch({type: Constants.updateProofState, payload: {username}})
     },
 
-    displayTrackStatement: (params: {sessionID: number, stmt: string}) => {
+    displayTrackStatement: params => {
     },
 
-    finishWebProofCheck: (params: {sessionID: number, rp: RemoteProof, lcr: LinkCheckResult}) => {
+    finishWebProofCheck: params => {
       const username = sessionIDToUsername[params.sessionID]
       dispatch(updateProof(params.rp, params.lcr, username))
       dispatch({type: Constants.updateProofState, payload: {username}})
       dispatch({type: Constants.decideToShowTracker, payload: {username}})
     },
-    finishSocialProofCheck: (params: {sessionID: number, rp: RemoteProof, lcr: LinkCheckResult}) => {
+    finishSocialProofCheck: params => {
       const username = sessionIDToUsername[params.sessionID]
       dispatch(updateProof(params.rp, params.lcr, username))
       dispatch({type: Constants.updateProofState, payload: {username}})
       dispatch({type: Constants.decideToShowTracker, payload: {username}})
     },
-    displayCryptocurrency: (params: {sessionID: number, c: Cryptocurrency}) => {
+    displayCryptocurrency: params => {
     },
-    displayUserCard: (params: {sessionID: number, card: UserCard}) => {
+    displayUserCard: params => {
       const username = sessionIDToUsername[params.sessionID]
       dispatch(updateUserInfo(params.card, username, getState))
     },
-    reportTrackToken: (params: {sessionID: number, trackToken: string}) => {
+    reportTrackToken: params => {
       const username = sessionIDToUsername[params.sessionID]
       const {trackToken} = params
       dispatch({type: Constants.updateTrackToken, payload: {username, trackToken}})
     },
-    confirm: (params: {sessionID: number, outcome: IdentifyOutcome}): bool => {
-      return false
+    confirm: params => {
```

Comment:
Spoke to max, he thinks we can do this as identify2 doesn't actually do this callback anymore